playbook/app/pb_kits/playbook/pb_table/styles/_sticky_header.scss

```diff
@@ -0,0 +1,23 @@
+@import "../../tokens/colors";
+
+[class^="pb_table"] {
+  &.sticky-header {
+    thead {
+      background: $white;
+      position: sticky;
+      top: -40px;
```

Comment:
I'm not sure about this. Are you setting this because of the nav. We might need a simple test with codesandbox CC @kre8sions

Reply:
Yea, the padding on the page-content bumps the header down by 40px this makes sure that the header sticks to the right place. And if the padding is different on specific pages I provided examples in the docs to get it to stick to the right place.

Reply:
Ok I'm with you. I don't want to ship a the kit with a fix for our docs. Other peoples sites may not have these issues. Lets fix the problem in playbook and keep top to `0`

[Set the top padding to `0`](https://github.com/powerhome/playbook/blob/master/playbook-website/app/javascript/site_styles/_site-style.scss#L87)
[Add `margin_top: "xl"` here](https://github.com/powerhome/playbook/blob/master/playbook-website/app/views/pages/kit_show.html.erb#L2)

^^^ These updates should allow you to set the top to 0

I'm going to have Jon reach out. We need to add more to the trouble shooting docs. This is a good writeup of dealing with most sticky issues: https://www.designcise.com/web/tutorial/how-to-fix-issues-with-css-position-sticky-not-working